utils/crawlGameSchedule.js

```diff
@@ -0,0 +1,54 @@
+const puppeteer = require("puppeteer");
+
+const crawlGameSchedule = async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  await page.goto("https://www.koreabaseball.com/Schedule/GameCenter/Main.aspx#none;");
+
+  const nextGameDate = await page.evaluate(() => (
+    document.querySelector(".date-txt").textContent
+  ));
+
+  const gameSchedule = await page.evaluate(() => {
+    const $gameList = document.querySelectorAll(".game-list > li");
+
+    const result = [];
+    for (let i = 0; i < $gameList.length; i += 1) {
```

Comment:
```suggestion
    const $gameList = document.querySelectorAll(".game-list > li");
    const result = [];
    
    for (let i = 0; i < $gameList.length; i += 1) {
```
저는 이렇게 쓰는 게 더 자연스러워 보이는데... 잘 모르겠네요 한솔님 의견은 어떤지도 들어봐야 될 것 같습니당

Reply:
저도 아래 빈 줄 넣는게 자연스러운 것 같아요!